traitsui/tests/editors/test_instance_editor.py

```diff
@@ -97,6 +97,29 @@ def _modify_inst_list(self, event):
     Item('change_options'),
     buttons=["OK"],
 )
+non_editable_droppable_view = View(
+    Item(
+        "inst",
+        editor=InstanceEditor(
+            editable=False,
+            droppable=True
+        ),
+        style='custom',
+    ),
+    buttons=["OK"],
+)
```

Comment:
I have no idea why one would want to do this... but it is a possible configuration one could make.  This would be when you are "editing" a single instance, but the single instance itself is not editable, so all that is displayed in a QLineEdit displaying the current instance.  The instance can be changed by dragging and dropping a different instance. The user can not select a different instance as theoretically there is no list to select from.